lib/inc/cpp-pcp-client/connector/v1/connector.hpp

```diff
@@ -106,6 +106,22 @@ class LIBCPP_PCP_CLIENT_EXPORT Connector : public ConnectorBase {
               uint32_t pong_timeouts_before_retry = 3,
               long ws_pong_timeout_ms = 5000);
 
+    // constructor for logging addition
+    Connector(std::vector<std::string> broker_ws_uris,
```

Comment:
Should there be another constructor similar to this one where the first arg is `std::string broker_ws_uri` (similar to the one on line 57)? All of the constructors follow that pattern of having one that accepts a single `broker_ws_uri` and one that accepts a vector of `broker_ws_uris`.

Reply:
I don't think it's actually necessary. I think the _original_ constructor just took a string and was left around for backward compatibility when we added the vector constructor to support multiple brokers. Then that pattern of string vs. vector was unnecessarily copied for the proxy and CRL constructors even though nothing was ever going to call the string version (since we had moved onto using a vector by that point).

It's basically the same reason I didn't add a constructor that just takes allows config for crl+logging or proxy+logging.